src/main/java/org/apache/maven/doxia/DefaultConverter.java

```diff
@@ -125,10 +120,172 @@
 
     private static final String MARKDOWN_SINK = "markdown";
 
-    /** Supported output format, i.e. supported Doxia Sink */
-    public static final String[] SUPPORTED_TO_FORMAT =
-        { APT_SINK, CONFLUENCE_SINK, DOCBOOK_SINK, FO_SINK, ITEXT_SINK, LATEX_SINK, MARKDOWN_SINK, RTF_SINK, TWIKI_SINK,
-            XDOC_SINK, XHTML_SINK, XHTML5_SINK };
+    /**
+     * All supported source formats
+     */
+    public enum ParserFormat
+    {
+        APT( APT_PARSER, "apt" ),
+        CONFLUENCE( CONFLUENCE_PARSER, "confluence" ),
+        DOCBOOK( DOCBOOK_PARSER, "xml", "article" ),
+        FML( FML_PARSER, "fml", "faqs" ),
+        TWIKI( TWIKI_PARSER, "twiki" ),
+        XDOC( XDOC_PARSER, "xml", "document" ),
+        XHTML( XHTML_PARSER, "html", "html" ),
+        XHTML5( XHTML5_PARSER, "html" ), // no autodetect support
+        MARKDOWN( MARKDOWN_PARSER, "md" );
```

Comment:
Wouldn't it make sense to move Doxia 2.0 first before handling removed format?

Reply:
We can easily do it in a subsequent PR. It is just enum constructors to remove...